app/src/main/java/emu/skyline/input/onscreen/OnScreenButton.kt

```diff
@@ -92,15 +100,25 @@ abstract class OnScreenButton(
     open fun render(canvas : Canvas) {
         val bounds = currentBounds
         val alpha = if (isPressed) (config.alpha - 130).coerceIn(30..255) else config.alpha
+        renderColors(drawable)
```

Comment:
Since color is only updated by the user, try setting the color to the drawable in an `init` block and provide a way to recreate the buttons when the user changes colors (it might not be needed depending on how `invalidate()` works), rather than setting the color at every render.